calico-cloud/image-assurance/scanners/cluster-scanner.mdx

```diff
@@ -0,0 +1,88 @@
+---
+description: Identify vulnerabilities in workloads to Kubernetes clusters.
+---
+
+# Automatically scan images for vulnerabilities using cluster scanner
+
+## Big picture
+
+Automatically scan Kubernetes workloads for vulnerabilities using {{prodname}} Image Assurance cluster scanner.
+
+## Value
+
+{{prodname}} Image Assurances cluster scanner helps you scan workloads running inside the cluster for vulnerabilities without much configuration. With Cluster Scanner you can:
+
+- Scan kubernetes workload images for vulnerabilities automatically as and when they are deployed.
+- View vulnerabilities for the workloads in the {{prodname}} management UI
+- Assess the impact of newly-found vulnerabilities and prioritize remediation efforts
+- Create exceptions to ignore specific vulnerabilities
+- Create alerts on high-severity vulnerabilities so you can delegate remediation efforts to the appropriate team
```

Comment:
I don't think we should include these last two bullet points (exceptions and alerts) here since they are separate features and aren't actually mentioned this page.

Reply:
See my other [comment here](https://github.com/tigera/docs/pull/676/files#r1209499128).

If this is a common boiler plate intro section, I think it should be the same text on all pages and also be links for the user to jump to the relevant part in the docs.

Reply:
We could possibly have a new `Getting Started` or `About Image Assurance` section for the boiler plate that can server as a landing page for Image Assurance. Lmk if yo want to do that.